classlib/src/main/java/org/teavm/classlib/java/util/TEnumMap.java

```diff
@@ -16,44 +16,34 @@
 package org.teavm.classlib.java.util;
 
 import java.io.Serializable;
-import java.util.AbstractMap;
-import java.util.AbstractSet;
 import java.util.Arrays;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.NoSuchElementException;
-import java.util.Objects;
-import java.util.Set;
+import org.teavm.classlib.java.lang.TCloneNotSupportedException;
+import org.teavm.interop.Rename;
 
-public class TEnumMap<K extends Enum<K>, V> extends AbstractMap<K, V> implements Serializable, Cloneable {
```

Comment:
Personally, I have another trend: to avoid using these `T`-prefixed classes when possible.

Reply:
Anyway, these changes make reviews harder

Reply:
This change was forced by considering specific TEnumMap case for putAll.